useful_functions.py

```diff
@@ -0,0 +1,22 @@
+"""file to properly format our dates"""
+
+
+def formation(date):
+    """Here will take the dateTime String, and format it in a way
+    that we like"""
+    fulldate = "%d/%d/%d" % (date.month, date.day, date.year)
+    return fulldate
+
+
```

Comment:
This function will filter the entries and only return the entries that contain the emotions we are looking for.